sdk/executor/src/main/java/com/mesosphere/sdk/executor/CustomExecutor.java

```diff
@@ -168,20 +165,20 @@ public void killTask(ExecutorDriver driver, Protos.TaskID taskId) {
         // TODO(mohit): Implement SIGKILL shutdown. Currently only perform SIGTERM.
         try {
             if (!launchedTasks.containsKey(taskId)) {
-                LOGGER.info("Unknown TaskId = {}. Cannot destroy something that is unknown.", taskId);
+                LOGGER.error("Unable to kill unknown TaskID: {}", TextFormat.shortDebugString(taskId));
```

Comment:
Going to miss the philosophical implications of the original phrasing, but this is probably for the best. 🤔